### PR TITLE
Chore: Update Wiki URL in issue_template

### PR DESCRIPTION
**Pull request by a76yyyy**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,6 @@ docs/.vitepress/cache
 
 # docs site build files
 docs/.vitepress/dist
+
+# local history
+.history
```
Comment (Dreamacro) [inline .gitignore:39]: What is this?

Comment (a76yyyy) [reply .gitignore:39]: Sorry, it's my personal custom.

Just a folder generated by a plug-in to record the local modification history of files

Comment (Dreamacro) [reply .gitignore:39]: It doesn't look like it's commonly used in development, so maybe this shouldn't be added

Comment (a76yyyy) [reply .gitignore]: Deleted

**Comment by Dreamacro** [comment]
thanks a lot

**Pull request merged**